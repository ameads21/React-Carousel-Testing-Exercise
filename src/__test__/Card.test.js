import { fireEvent, render } from "@testing-library/react";
import Card from "../Card";
import React from "react";

test("renders without crashing", function () {
  render(<Card />);
});

test("the snapshot of the page", function () {
  const { asFragment } = render(<Card />);
  expect(asFragment()).toMatchSnapshot();
});
