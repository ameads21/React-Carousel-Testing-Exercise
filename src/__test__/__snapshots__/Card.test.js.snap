// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`the snapshot of the page 1`] = `
<DocumentFragment>
  <div
    class="Card"
  >
    <h4
      class="Card-title"
    />
    <img
      class="Card-image"
    />
    <small
      class="Card-small"
    >
      Image  of .
    </small>
  </div>
</DocumentFragment>
`;
